web/src/main/java/com/epam/esm/web/exception/ControllerError.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.esm.web.exception;
+
+public class ControllerError {
```

Comment:
Could you rename all classes for Error in WEB ( it's not controller, but with this name can be misunderstanding)